index.html

```diff
@@ -0,0 +1,115 @@
+    <nav>
+      <div class="logo-container"><a href="index.html"><img src="assets/images/logo.png" id="logo" alt="logo image"></a>
+      </div>
+      <img src="assets/images/menu-icon.svg" id="menu-icon" alt="hamburger menu icon" class="icon">
+      <ul class="nav-links" id="nav-links">
+        <img src="assets/images/menu-close-Icon.png" id="menu-close-icon" class="menu-icon" alt="close menu icon">
+        <li><a href="about.html">About</a></li>
+        <li><a href="#">program</a></li>
+        <li><a href="#">Join</a></li>
+        <li><a href="#">Sponsor</a></li>
+        <li><a href="#">News</a></li>
+        <li><a href="#">Campaign</a></li>
+      </ul>
+    </nav>
+  </header>
+  <main id="main">
+    <section class="hero">
+      <div class="hero-text-content">
+        <span class="sub-title">Hello! Music Producers!</span>
+        <h1 class="title" id="title">Tritone Music conference 2023</h1>
+        <p class="description">A convention of like-minded music producers committed to the production of great music
+          whiles maintaining super standards.</p>
+        <span class="date">2023.01.16(Mon) ~ 25(Fri)</span>
+        <span class="venue">@The National theatre, Ghana.</span>
+      </div>
+    </section>
+    <section class="program">
+      <h1>Main Program</h1>
+      <div class="divider"></div>
+      <div class="programCardsContainer">
+        <article class="card">
+          <figure id="lecture-icon" class="card-icon"></figure>
+          <h2 class="activity">Lecture</h2>
+          <p class="card-description">Recieve invaluable information regarding the music industry, trends, brands,
+            standards, etc.</p>
+        </article>
+        <article class="card">
+          <figure id="listening-icon" class="card-icon"></figure>
+          <h2 class="activity">Listening Party</h2>
+          <p class="card-description">Get the opportunity to listen to some of the best music from the best producers that are yet to be released</p>
+        </article>
+        <article class="card">
+          <figure id="workshop-icon" class="card-icon"></figure>
+          <h2 class="activity">Workshop</h2>
+          <p class="card-description">Live hands-on workshop session to show you step by step, emerging trends, new gadgets, and how to use them</p>
+        </article>
+        <article class="card">
+          <figure id="giveaway-icon" class="card-icon"></figure>
+          <h2 class="activity">Give Aways</h2>
+          <p class="card-description">Get the chance to receive from a pool of $5M worth of gadgets.</p>
+        </article>
+        <article class="card">
+          <figure id="qna-icon" class="card-icon"></figure>
+          <h2 class="activity">QnA</h2>
+          <p class="card-description">Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur, esse.</p>
+        </article>
+      </div>
+      <a class="program-btn" href="">See the whole program</a>
+    </section>
+  </main>
+  <footer>
```

Comment:
- [ ] Dear @Reggeditt, I suggest you work on the `More`/`Less` feature in your spare time. I'm sure it's going to teach you a lot.

![Screenshot 2023-01-14 144443](https://user-images.githubusercontent.com/69077061/212470287-7f8caf20-34df-4dc1-a4a2-1e2b8da9e8c4.png)